TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -201,53 +201,52 @@ public static void OnCodeElement(CallStatement statement, CodeElementsParser.Cal
                 if (context.programNameOrProgramEntryOrProcedurePointerOrFunctionPointerVariable() == null)
                     DiagnosticUtils.AddError(statement, "Empty CALL is not authorized", context.Start);
 
-                foreach (var call in context.callUsingParameters()) CheckCallUsings(statement, call);
+                // Check each parameter of the CALL
+                foreach (var inputParameter in statement.InputParameters) CheckCallUsings(statement, inputParameter);
```

Comment:
Ok the foreach should go into the method itself as it is called `CheckCallUsings`.

Reply:
See https://github.com/TypeCobolTeam/TypeCobol/pull/1994/commits/6eaf2db4f3c76fb1dc27fecd27c5fb750c99ea19 .